article/app/views/fragments/inArticleAudio.scala.html

```diff
@@ -0,0 +1,59 @@
+
+<div class="inline-audio_container">
+    <div class="inline-audio_icon">
+    @fragments.inlineSvg("football-weekly-300", "journalism/in-article-audio-player")
+    </div>
+    <div class="inline-audio_content">
+        <div class="inline-audio_content_header">
+            <h3>
+                <span class="kicker">Listen / </span>
+                Class, schoolings and some harsh lessons - Football Weekly
+            </h3>
+        </div>
+        <div class="inline-audio_content_player">
+            <audio
+            class="inline-audio-player-element"
+            src="https://flex.acast.com/audio.guim.co.uk/2018/08/20-54476-gnl.fw.20180820.sj.fw2008.mp3"
+            preload="metadata"
+            data-media-id="gu-audio-5b7467c1e4b016cd4306acc7"
+            data-title="Class, schoolings and some harsh lessons – Football Weekly"
+            data-component="in-article audio"
+            >
+                <p>
+                    Sorry your browser does not support audio - but you can download here
+                    and listen https://flex.acast.com/audio.guim.co.uk/2018/08/20-54476-gnl.fw.20180820.sj.fw2008.mp3
+                </p>
+            </audio>
+            <div class="inline-audio_controls">
+                <div class="inline-audio_button">
+                @fragments.inlineSvg("btn-play", "journalism/in-article-audio-player")
+                </div>
+                <div class="inline-audio_time">
+                    <span id="inline-audio_time-played">00:00</span>
+                    <span> / </span>
+                    <span id="inline-audio_duration"> 1:19:44</span>
+                </div>
+            </div>
+            <div class="inline-audio_content_progress-bar">
+                <div class="played"></div>
+            </div>
+        </div>
+    </div>
+
+    <noscript>
+        <audio
+        controls
```

Comment:
Without this attribute, the one above will be invisible, right? Clever hack 👍 but let's not do that in the final version as the invisible audio will be caught by assistive tools.

Reply:
;) ah good to know